src/styles/blocks/card.scss

```diff
@@ -0,0 +1,87 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+
+  &__image {
+    display: flex;
+    margin-bottom: 40px;
+
+    img {
+      height: 134px;
+      width: 160px;
+      margin: 0 auto;
+    }
+  }
+
+  &__name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    text-align: left;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    font-weight: 400;
+    line-height: 14px;
+    text-align: left;
+    color: $secondary-text-color;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 16px 0 24px;
+
+    .rating-reviews {
+      line-height: 14px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    line-height: 18px;
+
+    .price-text {
+      font-size: 12px;
+      color: $secondary-text-color;
```

Comment:
The variable '$secondary-text-color' is used here as well. Make sure it is defined or imported correctly to prevent any issues during SCSS compilation.